packages/puppeteer-core/src/cdp/Puppeteer.ts

```diff
@@ -145,6 +145,10 @@ export class Puppeteer {
    * @returns Promise which resolves to browser instance.
    */
   connect(options: ConnectOptions): Promise<Browser> {
-    return _connectToCdpBrowser(options);
+    if (options.protocol === 'webDriverBiDi') {
+      throw new Error('Not implemented');
```

Comment:
```suggestion
      throw new UnsupportedOperation('Not implemented');
```

Do we need this PR given the next PR does the implementation?